### ISSUE TITLE
LibeRead refused to load a .epub file

### ISSUE THREAD

**Issue by MagnificentPako** [joyread/server#44]
I tried to upload an EPUB today but I got this
![](https://i.imgur.com/cyO1R5g.png)

Is this a problem with LibreRead or did I just create the epub the wrong way (I used Sigil)?

**Comment by MagnificentPako** [comment]
Here's a mega link for the file I most likely used 

https://mega.nz/#!UEo3XIIR!srItlParexCddekk2NljKUhKxHhhK_Qrghrlzn3yHXE

**Comment by mysticmode** [comment]
To put in detail. When the regex doesn't find the `<img` tag in the file, the code breaks. I need to put a condition when the cover image is not available, it should put a default cover.

**Comment by MagnificentPako** [comment]
There is no `<img>` tag for the cover? I'm pretty sure the _should_ be a cover file

**Comment by mysticmode** [comment]
Yes, there is. But there is a problem in the code on how it's fetching the cover file. It's a tricky hack actually which might not work in all cases. I will fix this soon.
